samples/Dressca/dressca-backend/src/Dressca.Web.Consumer/Controllers/BasketItemsController.cs

```diff
@@ -73,14 +73,16 @@ public async Task<IActionResult> GetBasketItemsAsync()
     {
         var buyerId = this.HttpContext.GetBuyerId();
 
-        var (basket, catalogItems) = await this.service.GetBasketItemsAsync(buyerId);
+        var (basket, catalogItems, deletedCatalogItemIds) = await this.service.GetBasketItemsAsync(buyerId);
 
         var basketResponse = this.basketMapper.Convert(basket);
         foreach (var basketItem in basketResponse.BasketItems)
         {
             basketItem.CatalogItem = this.GetCatalogItemSummary(basketItem.CatalogItemId, catalogItems);
         }
 
+        basketResponse.DeletedItemIds = (IList<long>)deletedCatalogItemIds;
```

Comment:
このキャストは型安全ではないので、以下のようにしたほうが良いです。

```suggestion
        basketResponse.DeletedItemIds = deletedCatalogItemIds.ToList();
```

`deletedCatalogItemIds` に入る件数は大半がゼロで、多くても1,2件ということを考えると、パフォーマンスを突き詰めてダイレクトキャストするより、型安全な方に倒しておいた方が良いと思います。

Reply:
型安全にキャストするよう修正しました。

Reply:
OKです。